conf/machine/include/tegra194.inc

```diff
@@ -39,3 +39,8 @@ L4T_DEB_SOCNAME = "t194"
 NVPMODEL ?= "nvpmodel_t194"
 NVPOWER ?= "jetsonpower_t194"
 NVFANCONTROL ?= "nvfancontrol_p2888"
+
+MACHINE_EXTRA_RRECOMMENDS += "kernel-module-tegra-udrm"
+
+KERNEL_MODULE_PROBECONF += "nvidia-drm"
+module_conf_nvidia-drm = "options tegra-udrm modeset=1"
```

Comment:
Should these be `tegra-udrm` for t194?

Reply:
Ah you are right ? 
I will fix it now 
Thanks  @kekiefer